packages/react-components/react-alert/src/components/Alert/useAlert.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import { Button } from '@fluentui/react-button';
+import {
+  CheckmarkCircle16Filled,
+  DismissCircle16Filled,
+  ErrorCircle16Filled,
+  Warning16Filled,
+} from '@fluentui/react-icons';
+import { getNativeElementProps, resolveShorthand } from '@fluentui/react-utilities';
+
+import type { AlertProps, AlertState } from './Alert.types';
+
+/**
+ * Create the state required to render Alert.
+ *
+ * The returned state can be modified with hooks such as useAlertStyles_unstable,
+ * before being passed to renderAlert_unstable.
+ *
+ * @param props - props from this instance of Alert
+ * @param ref - reference to root HTMLElement of Alert
+ */
+export const useAlert_unstable = (props: AlertProps, ref: React.Ref<HTMLElement>): AlertState => {
+  const { intent } = props;
+
+  /** Determine the icon to render based on the intent */
+  let iconToUse;
+  switch (intent) {
+    case 'success':
+      iconToUse = <CheckmarkCircle16Filled />;
+      break;
+    case 'error':
+      iconToUse = <DismissCircle16Filled />;
+      break;
+    case 'warning':
+      iconToUse = <Warning16Filled />;
+      break;
+    case 'info':
+      iconToUse = <ErrorCircle16Filled />;
```

Comment:
This should be "i" icon, not an exclamation mark.

Reply:
Spec
<img width="108" alt="image" src="https://user-images.githubusercontent.com/9615899/168740018-82a86905-bf9d-4228-8cb1-14da0fcf12c2.png">


Implementation
<img width="233" alt="image" src="https://user-images.githubusercontent.com/9615899/168739903-c581c8e7-eacc-4f9e-9865-8637e2e66930.png">

Reply:
Good catch. Updated